weed/server/filer_server.go

```diff
@@ -165,11 +165,14 @@ func NewFilerServer(defaultMux, readonlyMux *http.ServeMux, option *FilerOption)
 	if !util.LoadConfiguration("filer", false) {
 		v.SetDefault("leveldb2.enabled", true)
 		v.SetDefault("leveldb2.dir", option.DefaultLevelDbDir)
-		_, err := os.Stat(option.DefaultLevelDbDir)
-		if os.IsNotExist(err) {
-			os.MkdirAll(option.DefaultLevelDbDir, 0755)
+		if v.GetBool("leveldb2.enabled") {
```

Comment:
this is not related to this PR topic. Separate PRs will be easier to manage.

Reply:
removed